src/components/AppNavigation/AppNavigation.vue

```diff
@@ -41,43 +42,101 @@
 
     <li class="nav-item">
       <b-button v-b-toggle.access-control-menu variant="link">
-        <Password16 />
-        Access Control
+        <icon-access-control />Access Control
+        <icon-expand class="icon-expand" />
       </b-button>
-      <b-collapse tag="ul" id="access-control-menu">
+      <b-collapse tag="ul" id="access-control-menu" class="subnav">
         <b-nav-item href="javascript:void(0)">LDAP</b-nav-item>
-        <b-nav-item to="/access-control/local-user-management"
-          >Local user management</b-nav-item
-        >
+        <b-nav-item to="/access-control/local-user-management">
+          Local user management
+        </b-nav-item>
         <b-nav-item href="javascript:void(0)">SSL Certificates</b-nav-item>
       </b-collapse>
     </li>
   </b-nav>
 </template>
 
 <script>
-import Analytics16 from '@carbon/icons-vue/es/analytics/16';
-import DataCheck16 from '@carbon/icons-vue/es/data--check/16';
-import SettingsAdjust16 from '@carbon/icons-vue/es/settings--adjust/16';
-import Settings16 from '@carbon/icons-vue/es/settings/16';
-import Password16 from '@carbon/icons-vue/es/password/16';
+import IconAnalytics from '@carbon/icons-vue/es/analytics/16';
+import IconDataCheck from '@carbon/icons-vue/es/data--check/16';
+import IconSettingsAdjust from '@carbon/icons-vue/es/settings--adjust/16';
+import IconSettings from '@carbon/icons-vue/es/settings/16';
+import IconPassword from '@carbon/icons-vue/es/password/16';
+import IconChevronUp from '@carbon/icons-vue/es/chevron--up/16';
 
 export default {
   name: 'AppNavigation',
   components: {
-    Analytics16,
-    DataCheck16,
-    SettingsAdjust16,
-    Settings16,
-    Password16
+    iconOverview: IconAnalytics,
+    iconHealth: IconDataCheck,
+    iconControl: IconSettingsAdjust,
+    iconConfiguration: IconSettings,
+    iconAccessControl: IconPassword,
+    iconExpand: IconChevronUp
   }
 };
 </script>
 
-<style scoped>
-ul li ul {
+<style scoped lang="scss">
+.nav {
+  min-height: 100%;
+  padding-top: $spacer;
+}
+.subnav {
```

Comment:
`.subnav` works, but wondering  if a more semantic name might be `nav-item__list` or something similar?

Reply:
I figured since the top level `<ul>` has a classname of `nav` `.subnav` seems to make sense for this nested `<ul>`. if using BEM, then what about `.nav-item__nav` ? not sure if that makes sense either...

Reply:
Also there will be `nav-item` elements nested inside the `subnav`

Reply:
The nav-items are `<li>` elements and what you are styling is a `<ul>` which is why I thought nav-item__list. I was just looking for naming that easily reveals the expected hierarchy. Subnav didn't do that for me, but since the parent `nav` class is used on the top level `ul`, `.nav-item__nav` works well. :)

Reply:
Huh, I guess my thought is that subnav actually is a good and semantic classname, it's a commonly used and well defined pattern. 

According to nng:

> Definition: Subnavigation refers to the navigation UI that helps users access lower-level categories in the site’s information architecture (IA).

Also, helps us avoid BEM classnames getting too long/deeply nested.

Reply:
I don't mind updated to `.nav-item__nav` if you think that is more clear

Reply:
Thank you!